astropy/io/misc/pickle_helpers.py

```diff
@@ -9,7 +9,7 @@
 __all__ = ['fnpickle', 'fnunpickle']
 
 
-def fnunpickle(fileorname, number=0, usecPickle=True):
+def fnunpickle(fileorname, number=0):
```

Comment:
Should this be noted in the changelog? It's an API change even if it's a somewhat internal routine.

Reply:
It's in the public API so I think it should be deprecated. http://docs.astropy.org/en/stable/api/astropy.io.misc.fnunpickle.html

Reply:
yes, it's rather unfortunate that this one is public API at all. But since it's, I agree, we probably need to deprecate.